app/models/post.rb

```diff
@@ -1,16 +1,20 @@
 class Post < ApplicationRecord
-  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  belongs_to :author, class_name: 'User'
 
-  has_many :comments
-  has_many :likes
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
 
-  after_save :update_counter
+  validates :title, length: { in: 1..250 }, presence: true, allow_blank: false
+  validates_numericality_of :comments_counter, only_integer: true, greater_than_or_equal_to: 0
+  validates_numericality_of :likes_counter, only_integer: true, greater_than_or_equal_to: 0
 
-  def recent_comments
-    comments.order(created_at: :desc).limit(5)
-  end
+  after_save :update_posts_counter
 
-  def update_counter
+  def update_posts_counter
     author.increment!(:post_counter)
   end
+
+  def recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
```

Comment:
- [ ] Please add tests for this custom method here 😃 👍 this helps you to discover any error that can occur in it later 😃